### DISCUSSION TITLE
Permission error with v0.8.0

### DISCUSSION THREAD

**Discussion by DramaticPond** [dyc3/steamguard-cli#205] [Q&A] [answered]
After bfd0667, I always get a permission error.

*Command*
```
steamguard-cli -v trace
```

*Log*
```
INFO - reading manifest from /home/user/.config/steamguard-cli/maFiles
DEBUG - loading manifest: "/home/user/.config/steamguard-cli/maFiles/manifest.json"
INFO - Migrating manifest
DEBUG - deserializing manifest: version null
DEBUG - loading all accounts for migration
DEBUG - loading entry: "/home/user/.config/steamguard-cli/maFiles/steamuser.maFile"
INFO - Saving manifest and accounts...
DEBUG - saving steamuser.maFile
ERROR - Permission denied (os error 13)
```

*Info*
- OS: ArchLinux
- Arch: x84_64
- Kernel: Linux archlinux 6.3.9-zen1-1-zen
- Cargo version: 1.70.0

**Comment by dyc3** [comment, marked as answer]
This is not a steamguard-cli bug. There's something wrong with your file permissions.

What are the permissions on the `/home/user/.config/steamguard-cli/` directory? You need to have read and write permissions.

```bash
ls -l -R ~/.config/steamguard-cli/
```

You can set file permissions with chmod

```bash
chmod -R u+rw ~/.config/steamguard-cli/
```

You can set the file owner with chown
```bash
chown -R $USER:$USER ~/.config/steamguard-cli/
```

**Comment by DramaticPond** [comment]
You're right, it was an issue with the file permissions.
Changing the owner to my current user fixed it.

I moved the file from an old backup over to my new machine, thus they got screwed up.
I thought this was a problem with steamguard-cli due to it working prior to the commit.

Anyway, thanks for the help.
I'll check next time before opening an issue, sorry for the inconvenience.
